assets/lib/ul.css

```diff
@@ -0,0 +1,5 @@
+ul {
```

Comment:
why all `ul`s should have the same style?
you may have simple lists, like:

- item1
- item2
- item3

-